### PR TITLE
(fix) fixes for uncaught typeerror in order basket and added procedure price display

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

- fixed **Uncaught TypeError: (0 , w.closeWorkspace) is not a function** error on order basket
- added ability to show procedure prices on the order basket

## Screenshots
procedure

### CODE REVIEW

File: src/form/procedures-orders/add-procedures-order/procedures-order-form.component.tsx [hunk 1]
```diff
@@ -248,6 +258,42 @@ export function ProceduresOrderForm({
...
       >
         <div className={styles.form}>
+          <Tile id="" className={styles.prices}>
+            {billableItem ? (
+              <div className={styles.listContainer}>
+                <StructuredListWrapper isCondensed>
+                  <StructuredListHead>
+                    <StructuredListRow head>
+                      <StructuredListCell head className={styles.cell}>
+                        {t("mode", "payment methods")}
```
Comment (makombe) [inline src/form/procedures-orders/add-procedures-order/procedures-order-form.component.tsx:268]:
```suggestion
                        {t("paymentMethods", "Payment methods")}
```

File: src/form/procedures-orders/add-procedures-order/procedures-order-form.component.tsx [hunk 2]
```diff
@@ -248,6 +258,42 @@ export function ProceduresOrderForm({
...
+                      </StructuredListCell>
+                      <StructuredListCell head className={styles.cell}>
+                        {t("prices", "prices")}
```
Comment (makombe) [inline src/form/procedures-orders/add-procedures-order/procedures-order-form.component.tsx:271]:
```suggestion
                        {t("prices", "Prices")}
```

**Review by makombe — COMMENTED** [review]
Thanks @its-kios09  LGTM

File: src/form/procedures-orders/api.ts [hunk 3]
```diff
@@ -299,3 +315,21 @@ export function useConceptById(id: string) {
...
+  const billableItem = data?.data?.results?.find(
+    (item) => item?.concept?.uuid === billableItemId
+  );
+
+  return {
+    billableItem: billableItem,
+    isLoading: isLoading,
+    error,
+  };
+};
```
Comment (donaldkibet) [inline src/form/procedures-orders/api.ts:335]: We should not be handling billing information and work on the procedure repo. We should have an extension-slot then implement the logic of retrieving prices from esm-billing-app. something like this https://github.com/openmrs/openmrs-esm-patient-chart/blob/eea68c63846af39892bdf2e6bc1d08dc9fcc4824/packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order-form.component.tsx#L170 and logic is here
for lab and drug orders https://github.com/palladiumkenya/kenyaemr-esm-3.x/tree/main/packages/esm-billing-app/src/billable-services/billiable-item

I think Ted will also need to align with the design. The initial work wasn't informed on any designs

**Review by donaldkibet — CHANGES_REQUESTED** [review]
Thanks @its-kios09

**Comment by its-kios09** [comment]
Reviewed the code and created another PR

**Comment by ojwanganto** [comment]
@its-kios09 - Please avoid new PRs. We need to keep the review history.

**Pull request closed without merging**